src/frontends/paddle/tests/test_models/gen_scripts/generate_full.py

```diff
@@ -81,16 +81,18 @@ def full_shape_tensor(name : str, shape, dtype, value):
         outs = exe.run(
             fetch_list=[out])
 
-        saveModel(name, exe, feedkeys=[], fetchlist=[out], inputs=[], outputs=[outs[0]], target_dir=sys.argv[1])
+        saveModel(name, exe, feed_vars=[], fetchlist=[out], inputs=[], outputs=[outs[0]], target_dir=sys.argv[1])
 
     return outs[0]
 
 
 def full_shape_tensor_list(name : str, shape: list, dtype, value):
     paddle.enable_static()
     with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
-        node_shape = paddle.full(shape=[1], fill_value=shape, dtype='int32', name='shape')
-        x1 = paddle.full(shape=[2, node_shape], fill_value=value, dtype=dtype, name='full')
+        node_shape1 = paddle.to_tensor(3)
```

Comment:
Can the test case use the 'shape' parameters?

Reply:
of course, I changed it as bellow:

```
def full_shape_tensor_list(name : str, shape, dtype, value):
    paddle.enable_static()
    with paddle.static.program_guard(paddle.static.Program(), paddle.static.Program()):
        node_shape1 = paddle.to_tensor(shape)
        node_shape2 = paddle.to_tensor(2)
```

Reply:
> Can the test case use the 'shape' parameters?

done https://github.com/openvinotoolkit/openvino/pull/23010/commits/38c1e6d4be184c5c7522fe1a65f17b6b6f6bc2b6